src/config.rs

```diff
@@ -9,10 +9,19 @@ use ldap_poller::{config::TLSConfig, AttributeConfig, CacheMethod, ConnectionCon
 use serde::Deserialize;
 use url::Url;
 
+/// App prefix for env var configuration
+const ENV_VAR_CONFIG_PREFIX: &str = "FAMEDLY_LDAP_SYNC";
+
 impl Config {
-	/// Read the config from a file
-	pub async fn from_file(path: &Path) -> Result<Self> {
-		let config: Config = serde_yaml::from_slice(&tokio::fs::read(path).await?)?;
+	/// Create new config from file and env var
+	pub fn new(path: &Path) -> Result<Self> {
+		let config_builder = config::Config::builder()
+			.add_source(config::File::from(path))
+			.add_source(config::Environment::with_prefix(ENV_VAR_CONFIG_PREFIX).separator("__"));
```

Comment:
If I understand correctly, if [`config::Environment::list_separator`](https://docs.rs/config/latest/config/struct.Environment.html#method.list_separator) is not set, it's impossible to parse lists? The `config` documentation is a bit lacking, can you add some examples in the readme explaining how to configure via env vars? (especially `Vec` Parameters, including `Set<FeatureFlag>`).

Reply:
We should also add a test case for the one list arg we have (though I still wonder if we shouldn't just use a hexadecimal number for that setting).

Reply:
Indeed it was not possible to set lists with the code the way it was. Now it should work. I added tests that cover that part. 
Also, I encountered a small problem, if we try to set a list using empty a string on the env vars it fails. I couldn't solve that so I made added a default for this lists and for setting it as empty we just don't create the variable. There is a note on the readme about that.

Reply:
Hmmm, the value should still be mandatory in the config file. This is a bit awkward, is there really no better solution?

Reply:
I guess an empty list means no values end up marking a user as disabled, which is quite intuitive. Yeah, we can keep that behavior.

Reply:
If that list cannot be empty, then we should have used something like [`nonempty`](https://docs.rs/nonempty/latest/nonempty/) in the first place